next.clayui.com/src/components/clay/Select.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClaySelect, {ClaySelectWithOption} from '@clayui/select';
+import Editor from './Editor';
+import React from 'react';
+
+export const Select = () => {
+	const scope = {ClaySelect};
+	const code = `const Component = () => {
```

Comment:
Can we put this code reference all on the same line indentation? If need be, we can move this to a constant above `Select`

Reply:
See https://github.com/liferay/clay/pull/2061#discussion_r294540203